nerd-dictation

```diff
@@ -282,6 +282,21 @@ def simulate_typing_with_wtype(delete_prev_chars: int, text: str) -> None:
         ]
     )
 
+# -----------------------------------------------------------------------------
+# Simulate Input: STDOUT
+#
+def simulate_typing_with_stout(delete_prev_chars: int, text: str) -> None:
+    cmd = "stdout"
+
+    # No setup/tear-down.
+    if delete_prev_chars == SIMULATE_INPUT_CODE_COMMAND:
+        return
+
+    if delete_prev_chars:
+        sys.stdout.write("\x08"*delete_prev_chars)
```

Comment:
*picky* spaces around operators.

Reply:
Changes made.

For my reference, the setup is done with:
>nix build github:jtojnar/nixpkgs/nerd-dictation#nerd-dictation --cores 4

Running is done with
> nerd-dictation begin --timeout 5 --simulate-input-tool STDOUT